front/temporal/tracker/activities.ts

```diff
@@ -28,10 +36,14 @@ const TRACKER_WATCHED_DOCUMENT_MINIMUM_DIFF_LINE_LENGTH = 4;
 const TRACKER_WATCHED_DOCUMENT_MAX_DIFF_TOKENS = 4096;
 // The total number of tokens to show to the model (watched doc diff + maintained scope retrieved tokens)
 const TRACKER_TOTAL_TARGET_TOKENS = 8192;
-// The topK used for the semantic search against the maintained scope.
-// TODO(DOC_TRACKER): Decide how we handle this. If the top doc has less than $targetDocumentTokens,
-// we could include content from the next doc in the maintained scope.
-const TRACKER_MAINTAINED_DOCUMENT_TOP_K = 1;
+// The maximum number of chunks to retrieve from the maintained scope.
+const TRACKER_MAINTAINED_SCOPE_MAX_TOP_K = 8;
+
+// The size of the chunks in our data sources.
+// TODO(@fontanierh): find a way to ensure this remains true.
+const CHUNK_SIZE = 512;
+
+const TRACKER_SCORE_DOCS_MODEL_CONFIG = GPT_4O_MODEL_CONFIG;
```

Comment:
We can't use the model defined in the tracker config?

Reply:
No because I need those logprobs, we only have them on OAI and deepseek right now